libc/CMakeLists.txt

```diff
@@ -62,6 +62,24 @@ else ()
     optee/trace.c)
 endif ()
 
+if (ENABLE_FUZZING)
+  list(
+    APPEND
+    PLATFORM_SRC
+    ${MUSLSRC}/stat/lstat.c
```

Comment:
This will be enabled only when ENABLE_FUZZING flag is turned on. Fuzzer supoprt library handles syscalls via oe_register_syscall_hook(_syscall_hook); and overrides the syscall dispatching. So it will be an issue.